servers/republik/modules/crowdfundings/lib/scheduler/winbacks.js

```diff
@@ -0,0 +1,99 @@
+const debug = require('debug')('crowdfundings:lib:scheduler:winbacks')
+const moment = require('moment')
+const Promise = require('bluebird')
+
+const { send } = require('../mailLog')
+
+const {
+  PARKING_USER_ID
+} = process.env
+
+const DAYS_AFTER_CANCELLATION = 4
```

Comment:
(Same comments here as in `servers/republik/modules/crowdfundings/lib/scheduler/givers.js` about the naming.)